src/containers/Routes.tsx

```diff
@@ -16,7 +16,8 @@ function MainRoutes() {
         <Route path="/" element={<Home />} />
         <Route path="/register-organization" element={<OrgRegister />} />
         <Route path="/org-login" element={<Orglogin />} />
-        <Route path="/login-admin" element={<Adminlogin/>}/>
+        <Route path="/login-admin" element={<Adminlogin />} />
+        <Route path="/calendar" element={<Calendar />} />
```

Comment:
You can move this route to the dashboard routes

Reply:
I don't think it is a good idea. This is a component I created a route of it to let you be able to view it but the updated calendar component will not have a route for it. @samyshyaka